examples/heterogeneous-hello-world/main.cpp

```diff
@@ -63,103 +63,15 @@ struct gfx_DescriptorSet_0;
 struct gfx_PipelineState_0;
 
 bool executeComputation_0();
+extern unsigned char __computeMain[];
+extern size_t __computeMainSize;
```

Comment:
These names don't include the format of the generated code (e.g., `dxbc`), so there would be a conflict if the user specified two or more GPU target formats at once (`-target dxbc -target spirv`). It would be nice to clean that up in a follow-on.